docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md

```diff
@@ -70,6 +72,50 @@ Added `AttachmentsChanged` event to `Item`.
 
 **Available in**: Outlook for Windows (Office 365)
 
+---
+
+### Categories
+
+In Outlook, a user can group messages and appointments by using a category to color-code them. The user defines categories in a master list on their mailbox. They can then apply one or more categories to an item.
+
+#### [Categories](/javascript/api/outlook/office.categories)
+
+Added a new object that represents an item's categories.
+
+**Available in**: Outlook for Windows (Office 365)
```

Comment:
Should be in OWA React too let's confirm with bangalore

Reply:
For all categories APIs

Reply:
OWA React is not in PROD yet. When that occurs, I'll update the docs accordingly.